subscriber_topic: /temperature
max_power: 100000000.0
acc_lim_x: 1.0
acc_lim_y: 1.0
acc_lim_theta: 1.0
mecanum_chassis_controller:
  type: mecanum_chassis_controller/MecanumChassisController
  max_power: 100000000.0
  wheel_track: 0.4
  wheel_base: 0.4
  wheel_radius: 0.07625
  speed_mode: "global"
  linear_scale: 1.0
  angular_scale: 1.0
  pid_back_left:
    p: 10.0
    i: 0.0
    d: 0.2
    i_max: 0.0
    i_min: 0.0
    Target: 0.0
  pid_back_right:
    p: 10.0
    i: 0.0
    d: 0.2
    i_max: 0.0
    i_min: 0.0
    Target: 0.0
  pid_front_left:
    p: 10.0
    i: 0.0
    d: 0.2
    i_max: 0.0
    i_min: 0.0
    Target: 0.0
  pid_front_right:
    p: 10.0
    i: 0.0
    d: 0.2
    i_max: 0.0
    i_min: 0.0
    Target: 0.0

joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50